carbon/chain.json

```diff
@@ -140,9 +173,20 @@
       {
         "denom": "ibc/2B58B8C147E8718EECCB3713271DF46DEE8A3A00A27242628604E31C2F370EF5",
         "fixed_min_gas_price": 0,
-        "low_gas_price": 5e-05,
-        "average_gas_price": 5e-05,
-        "high_gas_price": 5e-05,
+        "low_gas_price": 0.00005,
+        "average_gas_price": 0.00005,
+        "high_gas_price": 0.00005,
+        "gas_costs": {
+          "cosmos_send": 10000000,
+          "ibc_transfer": 10000000
+        }
+      },
+      {
+        "denom": "ibc/2B9D089E73AC096934CD6BBBC71A1CD5DD2A4F8E94CC37CE7F861AC674044002",
```

Comment:
Error: Chain carbon does not have fee token ibc/2B9D089E73AC096934CD6BBBC71A1CD5DD2A4F8E94CC37CE7F861AC674044002 defined in its Assetlist.